drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -1670,6 +1670,7 @@ RID RenderingDeviceVulkan::texture_create(const TextureFormat &p_format, const T
 	// TODO: Check for support via RenderingDevice to enable on mobile when possible.
 
 #ifndef ANDROID_ENABLED
+	// TODO Find out if checking VK_KHR_IMAGE_FORMAT_LIST_EXTENSION_NAME is enough, we may be able to remove the platform check here.
```

Comment:
So we have to answer this question, does checking our extension fix this properly and can we thus remove the platform check here?

Reply:
There used to be a warning:
```
#ifndef _MSC_VER
#warning TODO check for support via RenderingDevice to enable on mobile when possible
#endif
```
I assume the check you have added is the necessary check.

Reply:
The warning is still there just a few lines above but its now in an android block. I've removed it all together, I think the extension check we now perform will do the trick. Needs to be tested on an android device that doesn't have this feature off course... :)